bcipy/parameters/symbol_positions.json

```diff
@@ -61,7 +61,7 @@
     ],
     "P": [
         0.53,
-        0.49
+        0.52
```

Comment:
We may be able to delete this file, it can be loaded from the data now.

Reply:
Only the NEU datasets were lacking this information, so I removed the json file and saved a copy for NEU datasets.